### DISCUSSION TITLE
[product] Basic Funding Dashboard

### DISCUSSION THREAD

**Discussion by iamronen** [opencollective/opencollective#5745] [Ideas]
This proposal originated with a need identified within OSC to make it possible for open-source funders to be able to look at their separate contributions to open-source as a  kind of "investment portfolio" that can help them to "see the whole" of their investment. 

The proposal has been posted as a markdown file in the opencollective repository:
https://github.com/opencollective/opencollective/blob/main/product/pitches/Basic%20Funding%20Dashboard.md

The conversation may lead to changes in the original proposal and may lead to other proposals altogether.

**Comment by iamronen** [comment]
attn:
OSC: @BenJam @RichardLitt @poohlaga 
OCF: @natehn @CarolineAWoolard @alanna

**Comment by BenJam** [comment]
In general, I'm all for much of what is here with a couple of caveats:

- I think we should permit people to become 'contributors' to Collectives without making a financial donation, this will allow us to track behaviours and allows companies to represent contributions like paying an employee or providing office space to a project. 
- I think we should do the above instead of providing functionality to record 'off ledger' transactions, I don't think they'll always model well (Tidelift only ever supports and entire package of projects for example)

## Next steps

Is there any reason why we shouldn't consider co-designing something like this with the likes of Indeed, AirBnB, Google and others?

## Interface

If I were nit-picking UI at this stage I would say (and feel free to ignore):

![FundingDashboard_Basic_Collapsed](https://user-images.githubusercontent.com/8337819/179219945-67a9f4bf-8519-4e51-b434-74473e002d40.png)

- I wonder if it would be better to show any projects/events and their target funding in here, mirroring the nested structure that the collectives create. We could *accent* new initiatives that were added since I last looked too. 
- I wonder whether we could incorporate updates into this as we anticipate this is likely to be the way Collectives share news and thanks of their success and failures (and Duane mentioned a thank you email from each project would be amazing as a start)

## Challenges

> Will this interface make sense if it shows up for organizations/collectives who do not make contributions? If not how should this be mitigated?

I think if we incorporate the idea above then yes, we provide for organisations to add a collective to this board, monitoring progress the project is making while they're making another kind of contribution. 

> How do we populate the "ecosystem" field?

By default, I would do this by the package manager, which is a good proxy for language and/or framework. As a CTO I might want to select between system level package managers (apt-get, yum etc) and application level package managers (ruby, node) too.

**Reply by poohlaga** [reply]
I like the idea of 'following' another collective if I don't sponsor them. e.g. I would follow a collective I would like to contribute to that does not show recent activity. If I see they become active again, I would become a paying sponsor.

**Comment by poohlaga** [comment]
In talking to a user/funder about this feature, I would like to suggest a way for them to use this using real-world examples. 

- Could you add a few specific use case examples at the beginning? Like I would suggest they create custom 'investment categories' (which look like tiers in the mockup) to monitor the health of projects and areas of interest. e.g. I would create one group for 'Brooklyn Mutual Aid' and another for 'NYC Arts and Cultural Spaces' if I was explaining this to a user outside of open source.
- How does the funder 'use' the data? Is it only in the OC interface or can they export it? Could I automate a webhook to send this page to me every month so I could review the past month and send it to other people in my organization who are not on OC but do want to see how we are spending our funds?

"Monthly" raise / spend is confusing because it is a 30-day rolling. If we do use a 30-day metric to calculate the numbers, we should rename it "Income (30-day)" "Expenses (30-day)".

If we only show the income under the "Recurring" using the recurring expenses feature we would miss any monthly recurring expenses that are made manually (e.g. like Braze does with GrapeJS). I don't know what percentage this on the platform is but it would just need to be something we explain in the docs.

In the dropdown, do the terms current and past recurring indicate the total $ of contributions from recurring payments? If so, we should be clear it does not indicate the status of contribution to these accounts (e.g. if they are active or inactive).

You use the term collapsed here but it looks expanded? "A collective can be collapsed to show more detailed information. The collapsed area has two sections:"

As an MVP, you could use categories to denote an ecosystem and this would make the reports more friendly for non open source funders

You mention "initially this should be rolled out in the admin-section of collectives and organizations who handle and contribute funds." We should also roll this out to Funds as well (example: https://opencollective.com/stripe or https://opencollective.com/season-of-docs)

**Reply by iamronen** [reply]
The pitch has been refined and updated to reflect the inputs and discussions around this feature.

Next up: 
 - [x] add the pitch for prioritization for design
 - [x] prepare a survey for collecting feedback from funders
 - [x] communicating the pitch to a few potential funders (including those who were interviewed in the research) to ask for feedback.
 - [x] Prepare survey for funder feedback
 - [x] communicating the pitch to the OC team

@BenJam and @RichardLitt please let me know about funders with whom you'd like to share this and ask for feedback (if I haven't spoken to them directly, I think it may be better for me to author the invitation, send it to you, and have you forward it to your contacts). My list includes:
 - [x] Chad Whitacre (Sentry)
 - [x] Mike Fix (Stripe)
 - [x] Joe Lencioni (Airbnb)
 - [x] Duane O'Brien (Indeed)
 - [x] Addy Osmani (Google Chrome)
 - [x] Mary Radomile & Amanda Casari (Google)
 - [x] Clare Dillon (Inner Source)

**Reply by RichardLitt** [reply]
Have you shared with each of these who've been checked off?

**Comment by iamronen** [comment]
I've created a (TypeForm) survey for collecting feedback from funders:
https://x7rwr9qad3h.typeform.com/to/UeJSP79N
